srtcore/epoll.cpp

```diff
@@ -659,9 +819,20 @@ int CEPoll::release(const int eid)
 
 int CEPoll::update_events(const SRTSOCKET& uid, std::set<int>& eids, const int events, const bool enable)
 {
+    // As event flags no longer contain only event types, check now.
```

Comment:
Added a sanity check for a case when someone would have wanted to subscribe to `SRT_EPOLL_ET`.